CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 5.0.2
+
+- Added: `function-disallowed-list` rule support to ban specific built-in functions (#422).
```

Comment:
In this repo, the changelog is not updated together with the PRs.I guess that this can be left here, and I can fix it later.